### DISCUSSION TITLE
Knowlg Friday's - Session 3

### DISCUSSION THREAD

**Discussion by vinukumar-vs** [Sunbird-Knowlg/Community#20] [General Discussions]
Folks,

Welcome to our 3rd session of Knowlg Fridays - A community connect event where we learn and grow together as a community!

[
![Knowlg Friday's Banner drawio](https://user-images.githubusercontent.com/4496096/173725935-2a9e8118-81e2-4821-9610-9dddd5deb2d7.png)
](https://docs.google.com/forms/d/1tN3UOmwuUSJYflX9IYAN_wfxwk4VsawFeofMPOELO18/viewform?edit_requested=true)

### Primary category

- What is the primary-category?
- What is the difference between content-type & primary-category?
- Advantages of primary-category over defining object-type?
- Define a schema for new primary-category(movie).
- Basic CRUD API's on the newly published schema(movie).

**Speakers:** @vinukumar-vs, @Jayaprakash8887 & @AmiableAnil 

Join webinar - [Zoom link](https://us06web.zoom.us/j/82109956783?pwd=bjltelN4MHk2eHdpV1lGT29hbFlKUT09)
Meeting ID: 821 0995 6783
Passcode: 002581

@pramodkvarma, @mathewjpallan, @pkranga, @Prerna789, @rahul-kankaria, @coolbung, @aks30, @vijiurs, @ChakshuGautam, @kiranharidas187, @pritamps, @chitranshu-keshav @rahul101001000 @coolbung
@SanthoshVasabhaktula, @vrayulu, @surendrasinghs, @pallakartheekreddy, @krgauraw, @luckynvdu @reshmi-nair, @sharathkashyap, @gandham-santhosh, @kameshbhr, @AmiableAnil, @Jayaprakash8887, @anandp504, @maheshkumargangula, @rhwarrier

**Comment by vinukumar-vs** [comment]
PPT Presentation of Knowlg friday's session-3

[Primary Category](https://docs.google.com/presentation/d/130zIDMayAMuFqHkhpAnWYwfVJXDgGLHcijsKQwLDNA0/edit?usp=sharing)

Session recording details:
https://us06web.zoom.us/rec/share/Ipc665CYGMvuMQ29nvqNDPCFYgAvaw7MdpHTLWZDByJqS8k5MrWIcLh1vhGucXoU.Rsk5eR8wP64xpAO4?startTime=1655458337000
Passcode: Sh#4Mm45

**Reply by pramodkvarma** [reply]
Is the recording of these sessions made available here? Please do. Thanks.

**Comment by joffinjoy** [comment]
@vinukumar-vs @Krishnaj20  Please share recorded session video link for this session.
